<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="PUN-6NLTH32" name="testscripts.scenario3.TC33" tests="1" failures="1" timestamp="9 Feb 2017 13:53:39 GMT" time="233.094" errors="0">
  <testcase name="runTC1" time="233.094" classname="testscripts.scenario3.TC33">
    <failure type="java.lang.AssertionError" message="Unable to find element with xpath == .//*[starts-with(@id,&#039;_FOpt1:_FOr1:0:_FOSritemNode_payables_payables_invoices&#039;)][contains(@id,&#039;:ifPopup::content&#039;)] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 60.12 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: &#039;3.0.1&#039;, revision: &#039;1969d75&#039;, time: &#039;2016-10-18 09:48:19 -0700&#039;
System info: host: &#039;PUN-6NLTH32&#039;, ip: &#039;10.32.150.98&#039;, os.name: &#039;Windows 7&#039;, os.arch: &#039;amd64&#039;, os.version: &#039;6.1&#039;, java.version: &#039;1.8.0_101&#039;
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:35943/, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss}]
Session ID: 7794e2ed-2c6e-4176-b12a-7fbc9c7979c2
*** Element info: {Using=xpath, value=.//*[starts-with(@id,&#039;_FOpt1:_FOr1:0:_FOSritemNode_payables_payables_invoices&#039;)][contains(@id,&#039;:ifPopup::content&#039;)]}">
      <![CDATA[java.lang.AssertionError: Unable to find element with xpath == .//*[starts-with(@id,'_FOpt1:_FOr1:0:_FOSritemNode_payables_payables_invoices')][contains(@id,':ifPopup::content')] (WARNING: The server did not provide any stacktrace information)
Command duration or timeout: 60.12 seconds
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.0.1', revision: '1969d75', time: '2016-10-18 09:48:19 -0700'
System info: host: 'PUN-6NLTH32', ip: '10.32.150.98', os.name: 'Windows 7', os.arch: 'amd64', os.version: '6.1', java.version: '1.8.0_101'
Driver info: org.openqa.selenium.ie.InternetExplorerDriver
Capabilities [{browserAttachTimeout=0, ie.enableFullPageScreenshot=true, enablePersistentHover=true, ie.forceCreateProcessApi=false, ie.forceShellWindowsApi=false, pageLoadStrategy=normal, ignoreZoomSetting=false, ie.fileUploadDialogTimeout=3000, version=11, platform=WINDOWS, nativeEvents=true, ie.ensureCleanSession=false, elementScrollBehavior=0, ie.browserCommandLineSwitches=, requireWindowFocus=false, browserName=internet explorer, initialBrowserUrl=http://localhost:35943/, javascriptEnabled=true, ignoreProtectedModeSettings=false, enableElementCacheCleanup=true, unexpectedAlertBehaviour=dismiss}]
Session ID: 7794e2ed-2c6e-4176-b12a-7fbc9c7979c2
*** Element info: {Using=xpath, value=.//*[starts-with(@id,'_FOpt1:_FOr1:0:_FOSritemNode_payables_payables_invoices')][contains(@id,':ifPopup::content')]}
	at org.testng.Assert.fail(Assert.java:94)
	at supportlibraries.DriverScript.wrapUp(DriverScript.java:560)
	at supportlibraries.DriverScript.driveTestExecution(DriverScript.java:104)
	at testscripts.scenario3.TC33.runTC1(TC33.java:29)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1129)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:746)
	at org.testng.TestRunner.run(TestRunner.java:600)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1264)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1189)
	at org.testng.TestNG.runSuites(TestNG.java:1104)
	at org.testng.TestNG.run(TestNG.java:1076)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:152)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:57)
]]>
    </failure>
  </testcase> <!-- runTC1 -->
</testsuite> <!-- testscripts.scenario3.TC33 -->
